data/Dockerfiles/dockerapi/Dockerfile

```diff
@@ -1,11 +1,14 @@
 FROM alpine:3.9
 LABEL maintainer "Andre Peters <andre.peters@servercow.de>"
 
-RUN apk add -U --no-cache python2 python-dev py-pip gcc musl-dev tzdata openssl-dev libffi-dev \
-  && pip2 install --upgrade pip \
-  && pip2 install --upgrade docker==3.0.1 flask flask-restful pyOpenSSL \
-  && apk del python-dev py2-pip gcc
+WORKDIR /app
```

Comment:
Is there a reason for this? :)

Reply:
What do you mean exactly?
This was my first Pull Reqest, so I had some problems with Github :-/ 
I had to delete the old respoistory, the current files are now here 

https://github.com/christianbur/mailcow-dockerized/blob/dockerapi_v3/data/Dockerfiles/dockerapi/Dockerfile

https://github.com/christianbur/mailcow-dockerized/blob/dockerapi_v3/data/Dockerfiles/dockerapi/server.py

Reply:
The WORKDIR. Not sure you really need it.

Can you create a new PR with the new files?